code/modules/clothing/clothing.dm

```diff
@@ -393,9 +394,20 @@ BLIND     // can't see anything
 	if(random_sensor)
 		//make the sensor mode favor higher levels, except coords.
 		sensor_mode = pick(0, 1, 1, 2, 2, 2, 3, 3)
-	adjusted = 0
+	adjusted = NORMAL_STYLE
 	..()
 
+/obj/item/clothing/under/equipped(mob/user, slot)
+	..()
+
+	if(mutantrace_variation && ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(DIGITIGRADE in H.dna.species.specflags)
+			adjusted = DIGITIGRADE_STYLE
+		else if(adjusted == DIGITIGRADE_STYLE)
+			adjusted = NORMAL_STYLE

```

Comment:
`code/modules/clothing/clothing.dm:407:error: : missing expression` 
happens in travis but not locally. A bit confused as to why.